Update secure-your-business-data.md (#987)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/security-and-compliance/secure-your-business-data.md

  
To set up multi-factor authentication:

> 1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Users** and **Active Users**.

Reviewer: ```suggestion
1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Users** > **Active Users**.
```

> 2. In the **Active Users** section, Click on  **multi-factor authentication**.

Reviewer: ```suggestion
2. In the **Active Users** section, click on  **multi-factor authentication**.
```

> 3. On the **Multi-factor authentication** page, select **user** if you are enabling this for one user Or you can peform a **Bulk Update**.

Reviewer: ```suggestion
3. On the **Multi-Factor Authentication** page, select **User** if you are enabling this for one user Or you can perform a **Bulk Update**.
```

> 4. Click on **Enable** under **Quick Steps**.

Reviewer 2: ```suggestion
4. Select **Enable** under **Quick Steps**.
```

> 4. CLick on **Enable** under **Quick Steps** 

Reviewer: ```suggestion
4. Select  **Enable** under **Quick Steps** 
```

> 5. In the Pop-up window, CLickc on **Enable Multi-Factor Authentication**.

Reviewer 2: ```suggestion
5. In the pop-up window, choose **Enable Multi-Factor Authentication**.
```

> 5. In the Pop-up window, CLickc on **Enable Multi-Factor Authentication**

Reviewer: ```suggestion
5. In the pop-up window, choose **Enable Multi-Factor Authentication**.
```
